pyintelowl/pyintelowl.py

```diff
@@ -102,9 +123,9 @@ def ask_analysis_availability(
 
     def send_file_analysis_request(
         self,
-        analyzers_requested: List[str],
         filename: str,
         binary: bytes,
+        analyzers_requested: List[str] = [],
```

Comment:
can u move this back between `self` and `filename` ? we shouldn't change the positions of arguments because that can break the code for a lot of people.